website/src/components/Changelog/GUI.tsx

```diff
@@ -0,0 +1,55 @@
+import Entry from "./Entry";
+import Wrapper from "./Wrapper";
+import Entries from "./Entries";
+
+export default function GUI({ type }: { type: string }) {
+  return (
+    <Wrapper
```

Comment:
So every new release means we'll have to re-format the `Wrapper` into an `Entry`?

Could they use the same syntax? e.g.

```xml
        <Wrapper version="1.0.8" date={new Date("2024-06-17")}>
          Fixes an issue in Windows that could cause the Wintun Adapter to fail
          to be created under certain conditions.
        </Wrapper>
```

The `title` would have to move but then there's no chance of messing up the previous release when adding a new one, it's just copy-paste and replacing the tag.

Reply:
Yeah, that was going through my head too. I'll see if I can make this simpler. I also thought using a plainer data structure (like JSON/XML) would make it easy to RSS feed this, but it's nice to support TSX in the `Entry` for Links and such...

Reply:
Managed to keep a single `Entry` component and conditionally render it based on the index of the Entries list.